api/types/client.go

```diff
@@ -202,10 +202,9 @@ type ImageBuildOutput struct {
 type BuilderVersion string
 
 const (
-	// BuilderV1 is the first generation builder in docker daemon
-	BuilderV1 BuilderVersion = "1"
-	// BuilderBuildKit is builder based on moby/buildkit project
-	BuilderBuildKit = "2"
+	BuilderUnset    BuilderVersion = ""  // BuilderUnset is unspecified (same behavior as default)
```

Comment:
nit: don't really like `BuilderUnset` public definition because no such version actually exists. Empty value exists only in the API. So:

```
if s == "" {
  v = types.BuilderV1
}
``` seems better in here